tests/unit/test_dbus_message.cpp

```diff
@@ -27,6 +27,8 @@
  */
 
 #include <assert.h>
+#include <string.h>
```

Comment:
```suggestion
#include <cassert>
#include <cstring>
```

Reply:
I'm not quite sure about this, does openthread prefer <xxx.h> to <cxxx> since I seldom see using the latter form in OpenThread code.
@bukepo @jwhui

Reply:
Let's use the OpenThread's style for now. @Moandor-y @gjc13